### ISSUE TITLE
AVT 1 - Data Table w/Toolbar - DAP violation when multiple toolbar regions are used in Web content

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4907]
## Environment
macOS 10.15.1
Chrome Version 78.0.3904.108 (Official Build) (64-bit)
DAP 1.8.0.0 Nov. 2019 Ruleset

## Description
This issue was reported on the [Accessibility Conformance Report landing page](https://www.ibm.com/able/product_accessibility/#/).
1. Go to the [ACR Landing page](https://www.ibm.com/able/product_accessibility/#/).
2. Select an offering
3. Run DAP and the following issues are present when multiple toolbars are used on the same page:
<img width="1218" alt="Screen Shot 2019-12-17 at 1 56 59 PM" src="https://user-images.githubusercontent.com/21676914/71030904-f80ed300-20d7-11ea-9123-7d32ba9c22cb.png">

To fix this issue. Remove the aria-label="data table toolbar" from the section.
